### PR TITLE
Horizontal Bar chart as Web component

### PR DESCRIPTION
**Pull request by Anush2303**
## New Behavior

1. Added Horizontal Bar Chart Absolute scale variant
2. Dark mode and RTL bug fixes for MultiStacked bar chart.

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts [hunk 1]
```diff
@@ -4,6 +4,97 @@ import { renderComponent } from '../helpers.stories.js';
 import { HorizontalBarChart as FluentHorizontalBarChart } from './horizontalbarchart.js';
 import { IChartDataPoint, IChartProps } from './horizontalbarchart.options.js';
 
+const hbcData = [
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts:7]: Call this singleBarHBC

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts [hunk 2]
```diff
@@ -98,10 +189,25 @@ export default {
 
 export const Basic: Story<FluentHorizontalBarChart> = renderComponent(storyTemplate).bind({});
 
+export const HBCAbsoluteScale: Story<FluentHorizontalBarChart> = renderComponent(html<
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts:192]: same here

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.ts [hunk 3]
```diff
@@ -187,6 +187,20 @@ export class HorizontalBarChart extends FASTElement {
   }
 
   public _createBarsAndLegends(data: IChartProps, barNo?: number) {
+    const _isRTL = this._isRTL;
+    const _computeLongestBarTotalValue = () => {
+      console.log(data);
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.ts:192]: remove console log

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.ts [hunk 4]
```diff
@@ -310,10 +323,10 @@ export class HorizontalBarChart extends FASTElement {
       .each(createBars)
       .on('mouseover', function (event, d) {
         const tooltipHTML = `
-        <div style="border-left:4px solid ${d.color}; padding-left: 8px;">
-            <div style="font-size: 15px;lineHeight: 16px;
-        color: theme.semanticColors.bodyText; margin-top: 4px;">${d.legend}</div>
-            <div style="font-weight:bold; color: ${d.color}; font-size: 30px; text-align: left; lineHeight: 36px; margin-top: 4px;">${d.data}</div>
+        <div style="border-left:4px solid ${d.color}; padding-left: 8px; height: 50px;">
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.ts:326]: too many style elements. Move them to css.

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts (hunk 1 above)
Comment (Anush2303) [reply packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts:7]: Okay

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts [hunk 5]
```diff
@@ -189,17 +189,16 @@ export default {
...
->`
+export const singleBarHBC: Story<FluentHorizontalBarChart> = renderComponent(html<StoryArgs<FluentHorizontalBarChart>>`
   <div>
-    <fluent-horizontalbarchart style="width: 100%" data="${JSON.stringify(hbcData)}"> </fluent-horizontalbarchart>
+    <fluent-horizontalbarchart style="width: 100%" data="${JSON.stringify(singleBarHBCData)}">
+    </fluent-horizontalbarchart>
   </div>
 `);
 
 export const HBCNMVariant: Story<FluentHorizontalBarChart> = renderComponent(html<StoryArgs<FluentHorizontalBarChart>>`
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts:199]: update this name also.

**Review by AtishayMsft — APPROVED** [review]

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts (hunk 5 above)
Comment (Anush2303) [reply packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts:199]: NMVariant, I am planning to take in separate PR

Comment (Anush2303) [reply packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts:199]: What should I rename this as ?

**Pull request merged**